game/scripts/npc/abilities/invoker_chaos_meteor.txt

```diff
@@ -19,7 +19,7 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastRange"                                    "700"
     "AbilityCastPoint"                                    "0.05"
-    "AbilityCooldown"                                     "35" //OAA
+    "AbilityCooldown"                                     "40" //OAA
```

Comment:
I would nerf the talent that increases meteor damage by a percentage, but maybe this is alright too.